gamemode/modules/logging/cl_init.lua

```diff
@@ -3,7 +3,32 @@ Log a message to console
 ---------------------------------------------------------------------------*/
 local function AdminLog(um)
 	local colour = Color(um:ReadShort(), um:ReadShort(), um:ReadShort())
-	local text = um:ReadString() .. "\n"
-	MsgC(Color(255,0,0), "[" .. GAMEMODE.Name .. "] ", colour, DarkRP.deLocalise(text))
+	local text = DarkRP.deLocalise(um:ReadString() .. "\n")
+	
+	MsgC(Color(255, 0, 0), "[" .. GAMEMODE.Name .. "] ", colour, text)
+	
+	hook.Call("darkRPLogPrinted", nil, text, colour)
 end
 usermessage.Hook("DRPLogMsg", AdminLog)
+
+/*---------------------------------------------------------------------------
+Interface
+---------------------------------------------------------------------------*/
+DarkRP.hookStub{
+	name = "darkRPLogPrinted",
```

Comment:
Should the D in `darkRP` be capitalised here? Existing hooks beginning with DarkRP have a capital D.

http://wiki.darkrp.com/index.php/Category:Hooks